### PR TITLE
Adding new exchange: SatoshiTango

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

File: dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Override
+    public String getUrl(int requestId, CheckerInfo checkerInfo) {
+        return URL;
+    }
+
+    @Override
+    protected void parseTickerFromJsonObject(int requestId, JSONObject jsonObject, Ticker ticker, CheckerInfo checkerInfo) throws Exception {
+        String pairId = checkerInfo.getCurrencyPairId();
+        String currencyCode = checkerInfo.getCurrencyCounterLowerCase() + "btc";
```
Comment (MaciejCiemiega) [inline dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java]: Please change this hardcoded `"btc"` to `checkerInfo.getCurrencyBaseLowerCase()`

File: dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Override
+    public String getUrl(int requestId, CheckerInfo checkerInfo) {
+        return URL;
+    }
+
+    @Override
+    protected void parseTickerFromJsonObject(int requestId, JSONObject jsonObject, Ticker ticker, CheckerInfo checkerInfo) throws Exception {
+        String pairId = checkerInfo.getCurrencyPairId();
```
Comment (MaciejCiemiega) [inline dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java]: `String pairId` is not used anywhere, can be removed

**Comment by pbosio** [comment]
@MaciejCiemiega harcoded string "btc" replaced by checkerInfo.getCurrencyBaseLowerCase() and removed some unnecessary lines

File: dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import com.mobnetic.coinguardian.model.Ticker;
+import com.mobnetic.coinguardian.model.currency.Currency;
+import com.mobnetic.coinguardian.model.currency.VirtualCurrency;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap; 
+
+public class SatoshiTango extends Market {
```
Comment (MaciejCiemiega) [inline dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java]: You also need to add this exchange at the end of the `MarketsConfig.java` file, like here https://github.com/mobnetic/BitcoinChecker/commit/68efb4aa085df653906860ae9c7486f2b46ca153

Comment (pbosio) [reply dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SatoshiTango.java]: Sorry, I thought I shouldn't commit MarketsConfig.
Exchange added at the end.

**Comment by MaciejCiemiega** [comment]
👍

**Pull request merged**